replay-tests/helloworld/app/controllers/AcceptPost.java

```diff
@@ -0,0 +1,16 @@
+package controllers;
+
+import play.mvc.Controller;
+import play.mvc.results.RenderJson;
+
+/** Used by LargePostBodySpec#exerciseFileChannelBufferWithLargePostBody() */
+public class AcceptPost extends Controller {
+
+  public RenderJson respondWithEmptyObject() {
+
+    // this triggers the log warning from TextParser.resetBodyInputStreamIfPossible
+    String body = request.params.get("body");
+
+    return new RenderJson("{}");
```

Comment:
How about include the length of the request's body in the response JSON?
E.g.:
```json
{
    "requestBodyLength": 9162
}
```

Then the `LargePostBodySpec` could do an assertion (`isEqualTo`, `isGreaterThan) for that on top of HTTP 200.

Reply:
Added that. Good idea indeed.

Reply:
Umm, sorry! I wanted to hint that `body` string variable's length, not just replying a header's value.

An "echo" service would be the best: that could be test with short and long requests.